lib/calendarium-romanum/util.rb

```diff
@@ -31,10 +32,31 @@ def initialize(year)
 
     # enumerates days of a month
     class Month < DateEnumerator
+      attr_reader :start
       def initialize(year, month)
         @start = Date.new year, month, 1
         @prop = :month
       end
     end
+
+    class DateParser
+      attr_reader :payload
```

Comment:
Please rename `#payload` to something more meaningful. E.g. something like `#dates` or `#date_range` would do well.

Reply:
Done. It's called `#date_range` now